### PR TITLE
Update `django.createsuperuser` command

### PR DESCRIPTION
**Pull request by MrShirokow**
- Add verbose names for email, username and password fields
- Update `django.createsuperuser` command according to new config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -378,6 +378,9 @@ Settings:
 * `default_superuser_email` default email of superuser (Default: `root@localhost`)
 * `default_superuser_username` default username of superuser (Default: `root`)
 * `default_superuser_password` default password of superuser (Default: `root`)
+* `verbose_email_name` verbose name for `email` field (Default: `Email address`)
```
Comment (TheSuperiorStanislav) [inline README.md:381]: Pls, add note that it should match with verbose name of user model

File: saritasa_invocations/_config.py [hunk 2]
```diff
@@ -90,6 +90,9 @@ class DjangoSettings:
             "password": "RDS_DB_PASSWORD",
         },
     )
+    verbose_email_name: str = "Email address"
```
Comment (TheSuperiorStanislav) [inline saritasa_invocations/_config.py:93]: Let's move it like this 
```python
verbose_email_name: str = ...
default_superuser_email: str  = ...
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ We follow [Semantic Versions](https://semver.org/).
 
 ## unreleased
 
+- Update `DjangoSettings` with new fields
+- Improve `django.createsuperuser` command
```
Comment (TheSuperiorStanislav) [inline CHANGELOG.md:8]: I think it should be one entry 

```suggestion
- Improve `django.createsuperuser` command with verbose name settings
```

File: README.md [hunk 4]
```diff
@@ -378,6 +378,15 @@ Settings:
...
+* `verbose_username_name` verbose name for `username` field (Default: `Username`)
+* `verbose_password_name` verbose name for `password` field (Default: `Password`)
+
+Note:
+
+* Values for `verbose_email_name`, `verbose_username_name`, `verbose_password_name`
+should match with verbose names of `User` model
```
Comment (TheSuperiorStanislav) [inline README.md:388]:
```suggestion
should match with verbose names of model that used [this setting](https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#substituting-a-custom-user-model)
```

File: saritasa_invocations/_config.py [hunk 5]
```diff
@@ -75,6 +75,9 @@ class DjangoSettings:
     migrate_command: str = "migrate"
     makemessages_params: str = "--all --ignore venv"
     compilemessages_params: str = ""
+    verbose_email_name: str = "Email address"
+    verbose_username_name: str = "Username"
+    verbose_password_name: str = "Password"
     default_superuser_email: str = "root@localhost"
     default_superuser_username: str = "root"
     default_superuser_password: str = "root"
```
Comment (TheSuperiorStanislav) [inline saritasa_invocations/_config.py:83]:
```suggestion
    verbose_email_name: str = "Email address"
    default_superuser_email: str = "root@localhost"
    verbose_username_name: str = "Username"
    default_superuser_username: str = "root"
    verbose_password_name: str = "Password"
    default_superuser_password: str = "root"
```

**Review by TheSuperiorStanislav — APPROVED** [review]

**Review by ron8mcr — APPROVED** [review]

**Pull request merged**